CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 # Copyright (c) 2019 Scipp contributors (https://github.com/scipp)
-cmake_minimum_required(VERSION 3.10.2)
+cmake_minimum_required(VERSION 3.12.0)
```

Comment:
Is this change required? Many of us are still on Ubuntu 18.04, which has cmake 3.10.2.